security/annobin-tests.py

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+#
+# Copyright: 2020 IBM
+# Author: Nageswara R Sastry <rnsastry@linux.vnet.ibm.com>
+
+import os
+from avocado import Test
+from avocado.utils import build, git, process
+from avocado.utils.software_manager import SoftwareManager
+
+
+class annobin(Test):
+    """
+    annobin-testsuite
+    :avocado: tags=security,testsuite
+    """
+    def setUp(self):
+        '''
+        Install the basic packages to support annobin
+        '''
+        # Check for basic utilities
+        smm = SoftwareManager()
+        deps = ['gcc', 'make', 'autoconf', 'gcc-plugin-devel', 'rpm-devel',
```

Comment:
Is the package name `gcc-plugin-devel` same for SuSE as well? I did not find this. Also not sure if `rpm-devel` would be available in debian based OS. Better to handle/exclude the distros which the test is not run on